packages/faustwp-cli/src/healthCheck/validateFaustEnvVars.ts

```diff
@@ -28,4 +28,28 @@ export const validateFaustEnvVars = () => {
       'Please make sure your production Faust app runs with a WordPress instance on https!',
     );
   }
+  if (getWpSecret()) {
+    // send secret key
+    const apiUrl = `${getWpUrl()}/wp-json/faustwp/v1/validate_secret_key`;
+    const headers = {
+      'x-faustwp-secret': getWpSecret() || '',
+    };
+    try {
+      const response = await fetch(apiUrl, {
+        headers,
+        method: 'POST',
+      });
+      if (response.status === 204) {
+        // Success: User receives a 204 status code
+      } else if (response.status === 401) {
```

Comment:
Since we do nothing here, can we just remove the 204 check and only check for 401?

Something like:
```
if ( response.status === 401 ) {
...
}